src/modules/emotes/emote.js

```diff
@@ -79,11 +79,23 @@ export default class Emote {
     }
     container.appendChild(image);
 
-    const tooltip = document.createElement('div');
-    tooltip.classList.add('bttv-tooltip', 'bttv-tooltip--up', 'bttv-tooltip--align-center');
-    tooltip.textContent = `${this.code}\n${
+    const enlargedImage = new Image();
+    enlargedImage.classList.add('bttv-emote-image-enlarged');
+    enlargedImage.src = createSrc(this.images, shouldRenderStatic, '4x');
+    enlargedImage.alt = `${prefixModifiers != null && prefixModifiers.length > 0 ? `${prefixModifiers.join(' ')} ` : ''}${
```

Comment:
i'd remove the alt text since it isn't really going to add value to a screen reader when they won't be able to open this tooltip